src/navigation/MainTabNavigator.js

```diff
@@ -132,7 +132,12 @@ const MainTabNavigator = createBottomTabNavigator(
     AboutStack
   },
   {
-    tabBarOptions: { activeTintColor: colors.accent, inactiveTintColor: colors.primary }
+    tabBarOptions: {
+      activeTintColor: colors.accent,
+      inactiveTintColor: colors.primary,
+      tabStyle: { marginTop: normalize(5) },
+      labelStyle: { margin: normalize(3) }
```

Comment:
where do you have these attributes from? is there a link to a documentation?
what is the difference between `tabStyle` and `labelStyle`?

Reply:
from official documentation:  https://reactnavigation.org/docs/3.x/bottom-tab-navigator/#tabbarlabel 
tabStyle = to style tabBar box 
labelStyle = to style just the text of tabBar 
in labelStyle there is a mistake , it should have a pudding not a margin (!)